tasks/el8/el8_base_deps.yml

```diff
@@ -100,6 +106,7 @@
     - 8080    # Candlepin Tomcat server
     - 8443    # ^
   when:
+    - "'services' in ansible_facts"
```

Comment:
the call to ansible_facts.services fails if systemd isn't there at all; so we check for it here